<html><body><h2>File (last 1,000 lines): /var/log/apx-hcc/apx-hcc-error.log</h2><h5>Page last updated on: 2014-08-23 17:21:00 UTC<br>Page last updated on: 2014-08-23 10:21:00 PST</h5><table><tr><td bgcolor=#CCCCCC><b>Critical(s):</b></td><td bgcolor=#FF0000><b>0</b></td></tr><tr><td bgcolor=#CCCCCC><b>Error(s):</b></td><td bgcolor=#FF0000><b>86</b></td></tr><tr><td bgcolor=#CCCCCC><b>Fatal(s):</b></td><td bgcolor=#FF0000><b>0</b></td></tr><tr><td bgcolor=#CCCCCC><b>Serialize(s):</b></td><td bgcolor=#FFFF00><b>0</b></td></tr><tr><td bgcolor=#CCCCCC><b>Warning(s):</b></td><td bgcolor=#FFFF00><b>0</b></td></tr></table><br>[2014-07-30T16:32:34] [ERROR] {'level': 'ERROR', 'unixtime': '1406737954574', 'app': {'app_user_info': {'session': 'h6idd4prlm9hwjiyvqeu7nq4hjh75lrh', 'user': u'', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'handled_by': u'django_middleware', u'message': u'exception while processing a view request', u'request_url': u'/account/login/', u'exception_message': u'Timeout(TimeoutError("HTTPSConnectionPool(host=\'hccint.apixio.com\', port=8443): Request timed out. (timeout=60.0)",),) is not JSON serializable', u'exception_name': u'TypeError'}}, 'component_name': 'frontend'}}, 'isotime': '2014-07-30T16:32:34.574745+00:00', 'source': 'hcc.middleware', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-07-30T18:39:56] [ERROR] {'level': 'ERROR', 'unixtime': '1406745596738', 'app': {'app_user_info': {'session': '9a4a3qd5bjpga3x82v88pm0h5wkstzwg', 'user': u'', 'user_agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.137 Safari/537.36'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'handled_by': u'django_middleware', u'message': u'exception while processing a view request', u'request_url': u'/account/login/', u'exception_message': u'Timeout(TimeoutError("HTTPSConnectionPool(host=\'hccint.apixio.com\', port=8443): Request timed out. (timeout=60.0)",),) is not JSON serializable', u'exception_name': u'TypeError'}}, 'component_name': 'frontend'}}, 'isotime': '2014-07-30T18:39:56.738867+00:00', 'source': 'hcc.middleware', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-07-30T22:32:26] [ERROR] {'level': 'ERROR', 'unixtime': '1406759546033', 'app': {'app_user_info': {'session': '9d4e9tkgqsfpqmtic14lzntorrpe8x7f', 'user': u'', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'handled_by': u'django_middleware', u'message': u'exception while processing a view request', u'request_url': u'/account/login/', u'exception_message': u'Timeout(TimeoutError("HTTPSConnectionPool(host=\'hccint.apixio.com\', port=8443): Request timed out. (timeout=60.0)",),) is not JSON serializable', u'exception_name': u'TypeError'}}, 'component_name': 'frontend'}}, 'isotime': '2014-07-30T22:32:26.033966+00:00', 'source': 'hcc.middleware', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-07-30T22:37:22] [ERROR] {'level': 'ERROR', 'unixtime': '1406759842620', 'app': {'app_user_info': {'session': '9d4e9tkgqsfpqmtic14lzntorrpe8x7f', 'user': u'', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'handled_by': u'django_middleware', u'message': u'exception while processing a view request', u'request_url': u'/account/login/', u'exception_message': u'Timeout(TimeoutError("HTTPSConnectionPool(host=\'hccint.apixio.com\', port=8443): Request timed out. (timeout=60.0)",),) is not JSON serializable', u'exception_name': u'TypeError'}}, 'component_name': 'frontend'}}, 'isotime': '2014-07-30T22:37:22.620910+00:00', 'source': 'hcc.middleware', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-07-30T22:47:47] [ERROR] {'level': 'ERROR', 'unixtime': '1406760467224', 'app': {'app_user_info': {'session': '9d4e9tkgqsfpqmtic14lzntorrpe8x7f', 'user': u'', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'handled_by': u'django_middleware', u'message': u'exception while processing a view request', u'request_url': u'/account/login/', u'exception_message': u'Timeout(TimeoutError("HTTPSConnectionPool(host=\'hccint.apixio.com\', port=8443): Request timed out. (timeout=60.0)",),) is not JSON serializable', u'exception_name': u'TypeError'}}, 'component_name': 'frontend'}}, 'isotime': '2014-07-30T22:47:47.224695+00:00', 'source': 'hcc.middleware', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-07-30T23:06:59] [ERROR] {'level': 'ERROR', 'unixtime': '1406761619397', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': 'g816i45h2i8rqam2ogovhqnjdluq4d8r', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u"could not retrieve a cohort for cohort_id={'cohort_id': 10000278, 'cohort_name': 'N/A'}, error=Cohort matching query does not exist."}}, 'component_name': 'frontend'}}, 'isotime': '2014-07-30T23:06:59.397512+00:00', 'source': 'sorting_hat_api.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-05T21:58:52] [ERROR] {'level': 'ERROR', 'unixtime': '1407275932509', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': '85pojf9kuf3j1i8vogl7lwin3zcqs0ry', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u'API error during sorting_hat.next_work_item', u'error': u'MaxRetryError("HTTPConnectionPool(host=\'10.191.15.150\', port=8085): Max retries exceeded with url: /document/4e17cc1c-0a9b-4cc9-943b-546ee48d231c/metadata (Caused by <class \'socket.error\'>: [Errno 111] Connection refused)",) is not JSON serializable'}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-05T21:58:52.509906+00:00', 'source': 'hcc_project.opportunity.api', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-05T22:01:05] [ERROR] {'level': 'ERROR', 'unixtime': '1407276065527', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': '85pojf9kuf3j1i8vogl7lwin3zcqs0ry', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u'API error during sorting_hat.next_work_item', u'error': u'MaxRetryError("HTTPConnectionPool(host=\'10.191.15.150\', port=8085): Max retries exceeded with url: /document/219be8eb-7449-4b24-885e-5e82eb78530e/metadata (Caused by <class \'socket.error\'>: [Errno 111] Connection refused)",) is not JSON serializable'}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-05T22:01:05.527461+00:00', 'source': 'hcc_project.opportunity.api', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-05T22:14:28] [ERROR] {'level': 'ERROR', 'unixtime': '1407276868184', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': '85pojf9kuf3j1i8vogl7lwin3zcqs0ry', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': '<failed_serde>', 'frontend': {'<failed_serde>': {'log_status': 'failed_log_conversion', 'message': '{"status": 400, "logger": "hcc-browser", "message": "Received error from the server while fetching a new opportunity", "browser": {"url": "https://hccint.apixio.com/#current_opportunity", "unixtime": 1407276064796, "isotime": "2014-08-05T15:01:04"}}'}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-05T22:14:28.184665+00:00', 'source': 'hcc_project.hcc.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-05T22:14:30] [ERROR] {'level': 'ERROR', 'unixtime': '1407276870936', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': '85pojf9kuf3j1i8vogl7lwin3zcqs0ry', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u'API error during sorting_hat.next_work_item', u'error': u'MaxRetryError("HTTPSConnectionPool(host=\'hccint.apixio.com\', port=8443): Max retries exceeded with url: /hcc/root@api.apixio.com/next_work_item/ (Caused by <class \'socket.error\'>: [Errno 111] Connection refused)",) is not JSON serializable'}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-05T22:14:30.936482+00:00', 'source': 'hcc_project.opportunity.api', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-05T22:14:44] [ERROR] {'level': 'ERROR', 'unixtime': '1407276884842', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': '85pojf9kuf3j1i8vogl7lwin3zcqs0ry', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': '<failed_serde>', 'frontend': {'<failed_serde>': {'log_status': 'failed_log_conversion', 'message': '{"status": 400, "logger": "hcc-browser", "message": "Received error from the server while fetching a new opportunity", "browser": {"url": "https://hccint.apixio.com/#current_opportunity", "unixtime": 1407276870332, "isotime": "2014-08-05T15:14:30"}}'}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-05T22:14:44.842547+00:00', 'source': 'hcc_project.hcc.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-05T22:14:49] [ERROR] {'level': 'ERROR', 'unixtime': '1407276889826', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': '85pojf9kuf3j1i8vogl7lwin3zcqs0ry', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u"could not retrieve a cohort for cohort_id={'cohort_id': 10000278, 'cohort_name': 'N/A'}, error=Cohort matching query does not exist."}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-05T22:14:49.826147+00:00', 'source': 'sorting_hat_api.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-05T22:14:56] [ERROR] {'level': 'ERROR', 'unixtime': '1407276896100', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': 'fpt2kjnl2885ghejoegmp9etpsau7csf', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u"could not retrieve a cohort for cohort_id={'cohort_id': 10000278, 'cohort_name': 'N/A'}, error=Cohort matching query does not exist."}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-05T22:14:56.100831+00:00', 'source': 'sorting_hat_api.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-07T03:43:17] [ERROR] {'level': 'ERROR', 'unixtime': '1407382997817', 'app': {'app_user_info': {'session': None, 'user': u'', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': '<failed_serde>', 'frontend': {'<failed_serde>': {'log_status': 'failed_log_conversion', 'message': '{"status": 400, "logger": "hcc-browser", "message": "Received error from the server while fetching a new opportunity", "browser": {"url": "https://hccint.apixio.com/#current_opportunity", "unixtime": 1407275931781, "isotime": "2014-08-05T14:58:51"}}'}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-07T03:43:17.817167+00:00', 'source': 'hcc_project.hcc.views', 'host': 'ip-10-197-35-234', 'client': '67.188.151.9'}<br>[2014-08-11T18:24:32] [ERROR] {'level': 'ERROR', 'unixtime': '1407781472523', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': 'kjkcyml191b5w9vy6vce0pi3pi144nin', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u"could not retrieve a cohort for cohort_id={'cohort_id': 10000278, 'cohort_name': 'N/A'}, error=Cohort matching query does not exist."}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-11T18:24:32.523900+00:00', 'source': 'sorting_hat_api.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-11T18:24:42] [ERROR] {'level': 'ERROR', 'unixtime': '1407781482145', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': 'kjkcyml191b5w9vy6vce0pi3pi144nin', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u"could not retrieve a cohort for cohort_id={'cohort_id': 10000278, 'cohort_name': 'N/A'}, error=Cohort matching query does not exist."}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-11T18:24:42.145283+00:00', 'source': 'sorting_hat_api.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-11T18:29:42] [ERROR] {'level': 'ERROR', 'unixtime': '1407781782764', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': 'kjkcyml191b5w9vy6vce0pi3pi144nin', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u"could not retrieve a cohort for cohort_id={'cohort_id': 10000278, 'cohort_name': 'N/A'}, error=Cohort matching query does not exist."}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-11T18:29:42.764575+00:00', 'source': 'sorting_hat_api.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br>[2014-08-11T18:29:48] [ERROR] {'level': 'ERROR', 'unixtime': '1407781788313', 'app': {'app_user_info': {'org_name': u'HealthCare Partners', 'session': 'w42yj8lxvkd5g2r6baw4rbsz8ssh1pv6', 'org_id': 10000291L, 'user': u'root@api.apixio.com', 'user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:31.0) Gecko/20100101 Firefox/31.0'}, 'app_name': 'hcc', 'hcc': {'error_name': u'exception', 'frontend': {u'exception': {u'message': u"could not retrieve a cohort for cohort_id={'cohort_id': 10000278, 'cohort_name': 'N/A'}, error=Cohort matching query does not exist."}}, 'component_name': 'frontend'}}, 'isotime': '2014-08-11T18:29:48.313288+00:00', 'source': 'sorting_hat_api.views', 'host': 'ip-10-197-35-234', 'client': '50.204.82.254'}<br></body></html>